source/Ship.h

```diff
@@ -194,6 +194,7 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// object is given the government's default will be used.
 	void SetHail(const Phrase &phrase);
 	std::string GetHail(std::map<std::string, std::string> &&subs) const;
+	const Phrase *GetHailPhrase() const;
```

Comment:
This and the isdefined seem to be common between this PR and core ship PR, how about opening a PR just to add those methods?

Reply:
if you reaaly want me to...

Reply:
It'd get merged quickly I think but dont feel forced ^^

Reply:
done